src/components/Article.js

```diff
@@ -102,7 +102,7 @@ const ArticleSection = ({
       class='article-section'
       ref={contentRef}
       style={imageUrl ? { backgroundImage: `url(${imageUrl})` } : {}}>
-      <div class='card'>
+      <div class='card' dir={dir}>
```

Comment:
Everything looks good to me, but a quick comment on adding 'dir' attributes. We usually try to always include both 'dir' *and* 'lang' attributes *if* we know that the direction is different than the given language. This will be more important when you separate out the UI language from the content language, but just putting this here for you to take into account.

The reason we do this on the web is accessibility -- screen readers will identify that a piece of text is in a different language, and adjust accordingly. Otherwise, they may misread the text if it's not tagged with a `lang=''` attribute.

This is probably more relevant to the UI fix,though, and is certainly not a merge blocker for this PR :)